### PR TITLE
Kinda got simulation working

### PR DESCRIPTION
**Pull request by AdityaMatam**
Added a bash script to install the necessary things to get the simulation working. Some modification to the installed files are required, and are specified in the instructions in the docs folder.

### CODE REVIEW

**Comment by Oluwatoni** [comment]
Why was this reopened?

**Comment by AdityaMatam** [comment]
My changes caused some problems that I didn't want merged while I was fixing them, so I closed it in the meantime. It's good now though.

**Comment by archielee** [comment]
Ok, so I want to set up CI first before I merge this in. I'll read it over though

File: Workspace/src/rover_autonomy/src/ball_tracking.py
Comment (archielee) [inline Workspace/src/rover_autonomy/src/ball_tracking.py:1]: Please don't merge this in yet, I don't think it works well right? Also, we're re-doing it

**Review by archielee — CHANGES_REQUESTED** [review]
Also, can you add a doc in the `docs/` directory

File: Workspace/src/rover_autonomy/src/fake_laser.py
Comment (archielee) [inline Workspace/src/rover_autonomy/src/fake_laser.py:1]: Do we still need fake_laser to fool the system?

File: Workspace/src/rover_autonomy/src/gps_navigation.cpp
Comment (archielee) [inline Workspace/src/rover_autonomy/src/gps_navigation.cpp:165]: Is this enabled atm? Or do we hardcode `ball_valid` somewhere

File: Workspace/src/rover_autonomy/src/gps_navigation.cpp
Comment (archielee) [inline Workspace/src/rover_autonomy/src/gps_navigation.cpp:204]: Don't use `ROS_WARN` for state changes, that's what `ROS_INFO` is for

File: Workspace/src/twist_mux/.gitignore
Comment (archielee) [inline Workspace/src/twist_mux/.gitignore:1]: We really shouldn't have individual .gitignores...

`twist_mux` can come in for now but I think it's going to be rewritten

File: docs/Simulation Instructions.txt
Comment (archielee) [inline docs/Simulation Instructions.txt:1]: This should be a Markdown file

**Review by archielee — CHANGES_REQUESTED** [review]

**Comment by archielee** [comment]
We should find a better place for the `husky_sim.rviz` and `setup_model.bash` files

**Pull request closed without merging**
